packages/calypso-e2e/src/hooks/suite.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Type dependencies
+ */
+import { Context } from 'mocha';
+
+/**
+ * Hook to start a new Browser instance.
+ *
+ * @param {Context} this Mocha context at the suite level.
+ * @returns {void} No return value.
+ */
+export function setMochaTimeout( this: Context ): void {
+	this.timeout( config.get( 'mochaTimeoutMS' ) );
```

Comment:
Are we actually going to change this value between environments? If not, we could just set the value in mocharc config file.

Reply:
Hmm, good point. I do not change timeout values between sections of the execution flow, indeed. 
I believe I already set the timeout value in the configuration value to something like 9000ms, but it should probably be a bit longer.